lib/ancestry/instance_methods.rb

```diff
@@ -38,7 +38,7 @@ def apply_orphan_strategy
           if self.ancestry_base_class.orphan_strategy == :rootify
             unscoped_descendants.each do |descendant|
               descendant.without_ancestry_callbacks do
-                descendant.update_attribute descendant.class.ancestry_column, (if descendant.ancestry == child_ancestry then nil else descendant.ancestry.gsub(/^#{child_ancestry}\//, '') end)
+                descendant.update_attribute descendant.class.ancestry_column, (if descendant.ancestry == child_ancestry then nil else descendant.ancestry.gsub(/^#{child_ancestry}#{Regexp.escape(self.ancestry_base_class.ancestry_delimiter)}/, '') end)
```

Comment:
line:38 had `ancestry_base_class.orphan_strategy`, so I just used that as a basis.

yea, consistency could be a little better